CONTRIBUTING.md

```diff
@@ -31,14 +32,37 @@ If your contribution is to add support for a new programming language, follow th
 6. Add the name of the new language to the `LanguageName` type
 7. Update `languageByName` in `src/language/language.ts`
 8. Update `src/tree-sitter-node/NodeParserAdapter.ts`
-9. Run tests
+9. Update `test/language/ExpressionBuilder.test.ts` to include the new language if it supports Cucumber Expressions
+10. Run tests
 
 As you are working on step 4 and 5 - use [tree-sitter playground](https://tree-sitter.github.io/tree-sitter/playground)
-to build your query. The queries must have [capturing nodes](https://tree-sitter.github.io/tree-sitter/using-parsers#query-syntax):
+to build your query. The queries _must have_ [capturing nodes](https://tree-sitter.github.io/tree-sitter/using-parsers#query-syntax):
 
 - `defineParameterTypeQueries`: `@expression`, `@name` and `@root`
 - `defineStepDefinitionQueries`: `@expression` and `@root`
 
+Changing the names of these aliases will result in test failures, even if they work within the tree-sitter playground.
+
+## Snippets
+
```

Comment:
We rely on the Cucumber Expressions library to generate *Cucumber Expressions* for undefined steps. We don't currently have a way to generate *Regular Expressions* for undefined steps.

For Cucumber implementations that don't (yet) support Cucumber Expressions, this means that the generated snippets will not work when they are run on that Cucumber implementation - it wouldn't know how to parse a Cucumber Expression.

It's still useful to generate snippets, but the snippet should have a comment saying something like:

> # This step definition uses a Cucumber Expression, which is currently not supported in [Cucumber implementation]. You must manually convert it to a Regular Expression so that [Cucumber implementation] can parse it.